packages/web-app-files/tests/unit/views/FilesDrop.spec.ts

```diff
@@ -1,140 +1,54 @@
-  // })
-  //
-  // it('should show spinner with loading text if wrapper is loading', () => {
-  //   const wrapper = getShallowWrapper({ loading: true })
-  //
-  //   expect(wrapper.find(selectors.loadingHeader).exists()).toBeTruthy()
-  //   expect(wrapper.find(ocSpinnerStubSelector).exists()).toBeTruthy()
-  //   expect(wrapper).toMatchSnapshot()
-  // })
-  //
-  // describe('when "loading" is set to false', () => {
-  //   const wrapper = getShallowWrapper()
-  //
-  //   it('should not show spinner and loading header', () => {
-  //     expect(wrapper.find(selectors.loadingHeader).exists()).toBeFalsy()
-  //     expect(wrapper.find(ocSpinnerStubSelector).exists()).toBeFalsy()
-  //   })
-  //
-  //   it('should show share information title', () => {
-  //     expect(wrapper).toMatchSnapshot()
-  //   })
-  //
-  //   it('should show vue drop zone with given options', () => {
-  //     const dropZone = wrapper.find('#files-drop-zone')
-  //     expect(dropZone.exists()).toBeTruthy()
-  //   })
-  //
-  //   it('should show error message if only it has truthy value', () => {
-  //     const wrapper = getShallowWrapper({
-  //       loading: false,
-  //       errorMessage: 'This is a test error message'
-  //     })
-  //
-  //     expect(wrapper).toMatchSnapshot()
-  //   })
-  // })
+  it('drop container always present', () => {
+    const { wrapper } = getMountedWrapper()
+    expect(wrapper.find('#files-drop-container').exists()).toBeTruthy()
+  })
+  describe('different files view states', () => {
+    it('shows the loading spinner during loading', () => {
+      const { wrapper } = getMountedWrapper({ loading: true })
+      expect(wrapper.find('oc-spinner-stub').exists()).toBeTruthy()
+    })
+    it('shows the "resource-upload"-component after loading', () => {
+      const { wrapper } = getMountedWrapper()
+      expect(wrapper.find('oc-spinner-stub').exists()).toBeFalsy()
+      expect(wrapper.find('resource-upload-stub').exists()).toBeTruthy()
+    })
+  })
 })
 
-function createStore(slogan = 'some slogan', davProperties = []) {
-  return getStore({ slogan, davProperties })
-}
-
-function getShallowWrapper({ store = createStore(), loading = false, errorMessage = null } = {}) {
-  return shallowMount(FilesDrop, {
-    localVue,
-    store,
-    mocks: {
-      $route,
-      $router: {
-        currentRoute: { name: 'some-route' },
-        resolve: (r) => {
-          return { href: r.name }
-        },
-        afterEach: jest.fn()
-      },
-      $uppyService: {
-        $on: jest.fn(),
-        useDropTarget: jest.fn(),
-        useXhr: jest.fn(),
-        subscribe: jest.fn()
-      }
-    },
-    setup: () => {
-      return {}
+function getMountedWrapper({ mocks = {}, loading = false } = {}) {
+  const defaultMocks = {
+    ...defaultComponentMocks({
+      currentRoute: { name: 'files-common-favorites' }
+    }),
+    $client: {
+      publicFiles: { list: jest.fn(() => Promise.resolve([{ getProperty: jest.fn() }])) }
```

Comment:
Maybe beyond the scope of this PR, but can't we use the client/clientService mock here?
Possibly even worth porting to web-client